plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/TypeHelper.java

```diff
@@ -208,6 +212,9 @@ public static Object getObject(Type type, RowResult row, int field)
         if (type instanceof VarbinaryType) {
             return Slices.wrappedHeapBuffer(row.getBinary(field));
         }
+        if (type.equals(DateType.DATE)) {
+            return row.getInt(field);
+        }
```

Comment:
Is this condition really needed?

Reply:
yes, it's. Otherwise, anyother type will be picked and that throw and exception. That my guess. isn't it ?

Reply:
I don't think the method takes DATE type.